app/lib/main/routing/routing.dart

```diff
@@ -0,0 +1,252 @@
+import 'package:acter/features/bug_report/pages/bug_report_page.dart';
+import 'package:acter/features/chat/pages/chat_page.dart';
+import 'package:acter/features/gallery/pages/gallery_page.dart';
+import 'package:acter/features/home/pages/dashboard.dart';
+import 'package:acter/features/search/pages/quickjump.dart';
+import 'package:acter/features/search/pages/search.dart';
+import 'package:acter/features/activities/pages/activities_page.dart';
+import 'package:acter/features/home/pages/home_shell.dart';
+import 'package:acter/features/news/pages/news_page.dart';
+import 'package:acter/features/onboarding/pages/login_page.dart';
+import 'package:acter/features/onboarding/pages/sign_up_page.dart';
+import 'package:acter/features/profile/pages/my_profile_page.dart';
+import 'package:acter/features/space/pages/overview_page.dart';
+import 'package:acter/features/space/pages/shell_page.dart';
+import 'package:acter/common/dialogs/dialog_page.dart';
+import 'package:acter/common/dialogs/error.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:go_router/go_router.dart';
+// ignore: implementation_imports
+import 'package:go_router/src/information_provider.dart';
+
+import 'package:acter/main/routing/routes.dart';
+
+final desktopPlatforms = [
```

Comment:
could remove this and import from utils constant (already defined) as this is common use-case for us.